### ISSUE TITLE
On the BLEU result 35.85

### ISSUE THREAD

**Issue by wyzjack** [juntang-zhuang/transformer-adabelief#2]
Hi, congrats on the nice work. But I have a problem in achieving your claimed result 35.85 in the paper. My run on my machine with cuda 9.0 and PyTorch 1.1 of Adabelief is 35.69.
![image](https://user-images.githubusercontent.com/36599296/117387978-6ac25d80-af1c-11eb-8135-e9b16b6191f3.png)
Could you please provide (link to) your model checkpoint file, or is there any other tricks in training? Thanks .

**Comment by juntang-zhuang** [comment]
Hi, I don't have pretrained weights now, it's too long ago and I perhaps deleted local files. It might be due to randomness, I'm not quite sure what did the AdaHessian authors change compared to original fairseq, and this repo is based on AdaHessian repo, it seems their result is higher than official fairseq, perhaps make this repo more sensitive to random seed.
